File: src/main/java/com/keyboardsamurais/apps/VoiceMessageBotApp.java
package com.keyboardsamurais.apps;

import com.keyboardsamurais.apps.notification.VoiceMessageBot;
import lombok.extern.slf4j.Slf4j;
import org.telegram.telegrambots.meta.TelegramBotsApi;
import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
import org.telegram.telegrambots.updatesreceivers.DefaultBotSession;

@Slf4j
public class VoiceMessageBotApp {

    public static void main(String[] args) {
        log.info("Starting VoiceMessageBotApp");

        try {
            var telegramBotsApi = new TelegramBotsApi(DefaultBotSession.class);
            VoiceMessageBot updatePostedHandler = new VoiceMessageBot();
            telegramBotsApi.registerBot(updatePostedHandler);
        } catch (TelegramApiException e) {
            throw new RuntimeException(e);
        }
    }
}

File: src/main/java/com/keyboardsamurais/apps/VoiceMessageBotApp.kt
package com.keyboardsamurais.apps

import com.keyboardsamurais.apps.config.EnvUtils
import com.keyboardsamurais.apps.notification.VoiceMessageBot
import mu.KotlinLogging
import org.telegram.telegrambots.bots.DefaultBotOptions
import org.telegram.telegrambots.meta.TelegramBotsApi
import org.telegram.telegrambots.updatesreceivers.DefaultBotSession


private val log = KotlinLogging.logger {}
object VoiceMessageBotApp {
    @JvmStatic
    fun main(ignoredArgs: Array<String>) {
        log.info("Starting VoiceMessageBotApp")

        val telegramBotsApi = TelegramBotsApi(DefaultBotSession::class.java)
        val options = DefaultBotOptions()
        val botToken = EnvUtils.getEnv("TELEGRAM_BOT_TOKEN")
        val updatePostedHandler = VoiceMessageBot(options, botToken)
        telegramBotsApi.registerBot(updatePostedHandler)

    }
}
